templates/watchguard_firebox_show_arp.template

```diff
@@ -0,0 +1,12 @@
+Value INTERFACE (\S+)
+Value IP ([0-9.]+)
+Value MAC (\S+)
+
+Start
+  ^\s*-- -> Start_record
```

Comment:
I'm not sure there is a need to go to a different state at this point.

I think we can just capture everything within **Start**
```
Start
  ^---
  ^---\s+ARP\s+Table
  ^---
  ^\?\s+\(${IP}\)\s+at\s+${MAC}\s+(\[\S+\]\s*)?([A-Z]+\s*)?on\s+${INTERFACE} -> Record
  ^\s*$$
  ^. -> Error
```